gym_compete/policy.py

```diff
@@ -155,9 +155,10 @@ def __init__(self, sess, ob_space, ac_space, n_env, n_steps, n_batch, hiddens=No
                          scope=scope, reuse=reuse, normalize=normalize)
         if hiddens is None:
             hiddens = [128, 128]
+        self.hiddens = hiddens
```

Comment:
Why does this need to be an instance variable? (Don't particularly object to it, just not sure I understand why it needs to be.)